Gems/AtomTressFX/Code/Rendering/HairFeatureProcessor.cpp

```diff
@@ -316,6 +316,62 @@ namespace AZ
                 return pass ? true : false;
             }
 
+            bool HairFeatureProcessor::CreateHairParentPass(RPI::RenderPipeline* renderPipeline)
+            {
+                if (HasHairParentPass(renderPipeline))
+                {
+                    CreatePerPassResources();
+                    return false;
+                }
+                const char* passRequestAssetFilePath = "Passes/AtomTressFX_PassRequest.azasset";
+                m_hairPassRequestAsset = AZ::RPI::AssetUtils::LoadAssetByProductPath<AZ::RPI::AnyAsset>(
+                    passRequestAssetFilePath, AZ::RPI::AssetUtils::TraceLevel::Warning);
+                const AZ::RPI::PassRequest* passRequest = nullptr;
+                if (m_hairPassRequestAsset->IsReady())
+                {
+                    passRequest = m_hairPassRequestAsset->GetDataAs<AZ::RPI::PassRequest>();
+                }
+                if (!passRequest)
+                {
+                    AZ_Warning("AtomTressFx", false, "Failed to load PassRequest from %s", passRequestAssetFilePath);
```

Comment:
What will happen in this case?  this is an example where on very rare cases the asset might not be ready on time - can this lead to a crash / Assert?
will it attempt to correct it over next frame if it wasn't loaded, and will the current frame still work properly?

Reply:
I changed it to AZ_Error and updated the log message